doc/README.md

```diff
@@ -3,7 +3,7 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires a few hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires several hundred gigabytes or more of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to several days or more.
```

Comment:
nit: maybe just say hundreds
```suggestion
Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires hundreds of gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to several days or more.
```

Reply:
I think I prefer "several hundred or more", as given the current size required, it conveys the floor to expect a bit more, and when the requirement breaks past a terabyte, "or more" allows more time before updating again, whereas "hundreds" might be construed as or implies "not thousands."

Reply:
> whereas "hundreds" might be construed as or implies "not thousands."

I just wanted to be specific on the what is the requirement currently. 
"several hundred or more" is not specific.

> it conveys the floor to expect a bit more, and when the requirement breaks past a terabyte, "or more" allows more time before updating again.

Fair enough, it is likely to break hundreds, so this comment can be ignored; it's a non-blocking nit.